aries-site/src/pages/components/footer.js

```diff
@@ -18,20 +23,155 @@ const Footer = () => (
+          secondary pages that may not be part of the{' '}
+          <Anchor label="Header" href={nameToPath('Header')} /> navigation
+          structure.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Usage" level={3} gap="small">
+        <SubsectionText>
+          A footer is a consistent element to be used at the bottom of all of
+          your applications or webpages.
+        </SubsectionText>
+        <SubsectionText>
+          The most common footer is an{' '}
+          <Anchor label="App Footer" href="#app-footer" /> which is contains
+          legal and other important resources that pertain to the entirety of an
+          application.
+        </SubsectionText>
+        <SubsectionText>
+          The other type of footer is a{' '}
+          <Anchor label="Page Footer" href="#page-footer" /> which contains
+          additional actions or resources specific to a single page of an
+          application.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="App Footer" level={3}>
+        <SubsectionText>
+          A simple, clean and straightforward app footer displays useful
+          information for the user that couldn’t be displayed on the header.
+          This Footer is typically present on every screen of an application and
+          its contents are universal to the entire application.
+        </SubsectionText>
+        <Text weight="bold">
+          Always provide these essential elements in your app footer:
+        </Text>
+        <BulletedList
+          items={[
+            'Copyright information',
+            'Terms of Use',
+            'Privacy',
+            'Security',
+          ]}
+        />
+        <SubsectionText>
+          Check out this <Anchor href="#" label="simple Footer example" /> to
+          see how these essential elements are included within a Footer.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Page Footer" level={3}>
+        <SubsectionText>
+          A page-level Header provides additional resources or functions for a
+          single page within an application.
+        </SubsectionText>
+        <Text weight="bold">A page-level Header should:</Text>
+        <BulletedList
+          items={[
+            'always be used in addition to the App Footer',
+            'be placed above the App Footer in the layout',
+            'contain actions relevant to a single page of an app',
+          ]}
+        />
+        <SubsectionText>
+          For more guidance, check out an example of{' '}
+          <Anchor
+            label="how to use a page-level footer"
+            href="#footer-for-a-single-page"
+          />
+          .
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <SubsectionText>
+          It is wrap buttons or other elements at the bottom of a page in a
```

Comment:
Footer links are wrapped in buttons or other...

Reply:
Whoops, grammar got lost here. Updating to clarify!